### PR TITLE
docs(README): Add part about the cli

### PR DESCRIPTION
**Pull request by korthout**
I've updated the README's section on Usage to also discuss using the `init` command and I've added a part about the command-line interface.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,11 +61,20 @@ module.exports = function(config){
...
 
-As you can see, the config file is *not* a simple JSON file, it should be a common js (a.k.a. npm) module.  
-You might recognize this way of working from the karma test runner.
+As you can see, the config file is *not* a simple JSON file. It should be a common js (a.k.a. npm) module. You might recognize this way of working from the karma test runner.
 
 Make sure you *at least* specify the `files` and the `testRunner` options when mixing the config file and/or command line options.
 
+## Command-line interface
+
+Stryker can also be installed, configured and run using the [Stryker-CLI](https://github.com/stryker-mutator/stryker-cli). If you plan on using Stryker in more projects, the Stryker-CLI is the easiest way to install, configure and run Stryker for your project.
```
Comment (nicojs) [inline README.md]: Maybe we can add some information here about the working of `stryker-cli`. Mainly that it is only used as a proxy to the local `stryker` instance.

File: README.md [hunk 2]
```diff
@@ -59,11 +61,20 @@ module.exports = function(config){
 }
 ```
 
-As you can see, the config file is *not* a simple JSON file, it should be a common js (a.k.a. npm) module.  
-You might recognize this way of working from the karma test runner.
+As you can see, the config file is *not* a simple JSON file. It should be a common js (a.k.a. npm) module. You might recognize this way of working from the karma test runner.
```
Comment (nicojs) [inline README.md]: It's actually not called an 'npm' module, but a 'node' module. :) just realize that now.

**Comment by korthout** [comment]
@nicojs I've made the requested changes. Please let me know how you feel about them.

**Comment by korthout** [comment]
I've also updated the README of the `stryker-cli` repo. This last commit is ment to make the descriptions for usage of the CLI more friendly and the same for both repos.

**Comment by korthout** [comment]
@nicojs I think this is ready to be merged. Can you take a final look?

**Pull request merged**